examples/federated_statistics/image_stats/image_stats_job/custom/image_histogram.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import os
+from typing import Dict, List
+
+import numpy as np
+from monai.data import ITKReader, load_decathlon_datalist
+from monai.transforms import LoadImage
+
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.statistics_spec import Statistics, Histogram, Feature, DataType, Bin, HistogramType
+
+
+class ImageHistogram(Statistics):
```

Comment:
When call this ImageHistogram I will think of it is a subclass of Histogram.
Subjectively I suggest change to "ImageStatistics"